packages/ui/src/components/MaterialsCard/MaterialsCard.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Box, BoxProps } from '@chakra-ui/react';
+
+import { Button } from '../Button';
+import { Typography } from '../Typography/Typography';
+
+type MaterialsCardStatus = 'idle' | 'loading' | 'generated';
+
+export interface MaterialsCardProps extends BoxProps {
+  status: MaterialsCardStatus;
+}
+
+type MaterialsCardOptions = { [status: string]: { statusText: string; buttonText: string } };
+
+const MaterialsCardText: MaterialsCardOptions = {
+  idle: {
+    statusText: 'Kliknij przycisk poniżej, aby wygenerować swoja unikalną listę z materiałami.',
+    buttonText: 'Wygeneruj listę',
+  },
+  loading: {
+    statusText: 'Trwa generowanie materiałów...',
+    buttonText: 'Generowanie listy...',
+  },
+  generated: {
+    statusText: 'Twoja lista materiałów znajduje się na platformie Process.st',
+    buttonText: 'Przejdź do listy materiałów',
+  },
+};
+
+export const MaterialsCard = ({ status = 'idle', ...props }) => {
+  return (
+    <Box status={status} w="100%" boxShadow="base" borderRadius="8px" padding="24px" {...props}>
```

Comment:
🤔 I'm not sure the Box component can receive a `status` prop. Why did you need to pass it here?